credentials/prc/credential.json

```
@@ -1,7 +1,8 @@
 {
   "@context": [
     "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/citizenship/v1"
+    "https://w3id.org/citizenship/v1",
+	  "https://www.w3.org/ns/credentials/examples/v2"
```

```suggestion
    "https://w3id.org/citizenship/v1",
    "https://www.w3.org/ns/credentials/examples/v2"
```

Sneaking tab hiding in there.